utils/terrascript_client.py

```diff
@@ -601,6 +601,27 @@ def populate_tf_resource_s3(self, resource, namespace_info):
         if common_values.get('lifecycle_rules'):
             # common_values['lifecycle_rules'] is a list of lifecycle_rules
             values['lifecycle_rule'] = common_values['lifecycle_rules']
+        if versioning:
+            lrs = values.get('lifecycle_rule', [])
```

Comment:
should this be `lifecycle_rules`?

Reply:
It's lifecycle_rules in common_values, but just lifecycle_rule in values.  In terraform, it's multiple lifecycle_rule definition stanzas for an s3 object.

Reply:
perfect!